src/object_store/Cargo.toml

```diff
@@ -37,6 +37,10 @@ thiserror-ext = { workspace = true }
 tokio = { version = "0.2", package = "madsim-tokio", features = ["fs"] }
 tokio-retry = "0.3"
 tracing = "0.1"
+
+[patch.crates-io]
+reqsign = { git = "https://github.com/wcy-fdu/reqsign.git", rev = "f6ff677" }
```

Comment:
Why is the `rev` different from the one specified in Cargo.toml?

Reply:
discussed offline.